ansible/roles/tensorrt/tasks/main.yaml

```diff
@@ -32,3 +25,6 @@
     - libnvonnxparsers-dev
     - libnvparsers8
     - libnvparsers-dev
+    - libnvinfer-bin
+    - libnvinfer-samples
+    - tensorrt
```

Comment:
Isn't it enough to hold only `tensorrt`?
https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html

Reply:
You're right. I held only `tensorrt` at https://github.com/autowarefoundation/autoware/pull/2766/commits/8ce41112eeb88718637bf07c5ff8199e7eb6e2b0.